src/js/action-types.js

```diff
@@ -16,11 +16,35 @@ export const SET_UPLOAD_DATA_TYPES = 'SET_UPLOAD_DATA_TYPES'
 export const SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME = 'SET_UPLOAD_DATA_TYPE_BY_HEADER_NAME'
 export const SET_VISUALIZATION = 'SET_VISUALIZATION'
 
+export const DASHBOARD_CREATE_VISIBILITY = 'DASHBOARD_CREATE_VISIBILITY'
+export const DASHBOARD_CREATE_SUBTITLE_FORM = 'DASHBOARD_CREATE_SUBTITLE_FORM'
+export const DASHBOARD_CREATE_TITLE_FORM = 'DASHBOARD_CREATE_TITLE_FORM'
+export const DASHBOARD_CREATE_SAVE = 'DASHBOARD_CREATE_SAVE'
+export const DASHBOARD_DELETE_VISIBILITY = 'DASHBOARD_DELETE_VISIBILITY'
+export const DASHBOARD_EDIT_POPULATE_FORM = 'DASHBOARD_EDIT_POPULATE_FORM'
+export const DASHBOARD_EDIT_SUBTITLE_FORM = 'DASHBOARD_EDIT_SUBTITLE_FORM'
+export const DASHBOARD_EDIT_TITLE_FORM = 'DASHBOARD_EDIT_TITLE_FORM'
+export const DASHBOARD_EDIT_VISIBILITY = 'DASHBOARD_EDIT_VISIBILITY'
+export const DASHBOARD_EDIT_SAVE = 'DASHBOARD_EDIT_SAVE'
+export const DASHBOARD_SELECT = 'DASHBOARD_SELECT'
```

Comment:
These are all actions and generally they would make more sense when reading them to start with actions instead of nouns. Almost all of these would read better and make more sense with DASHBOARD coming after an action.